### PR TITLE
centralize prompt: setup.py

### PR DESCRIPTION
**Pull request by Eggrolling-hu**
### PR Quality Checklist
- [ ] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [ ] I have considered potential risks and mitigations for my changes.
- [ ] I have documented my changes clearly and comprehensively.
- [ ] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: autogpt/prompts/default_prompts.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    - Proactively take the lead in guiding you and offering suggestions when faced with unclear information or uncertainty to ensure your marketing strategy remains on track.
+    """)
+
+TASK_PROMPT_AICONFIG_AUTOMATIC = (
+    "Task: '{{user_prompt}}'\n"
+    "Respond only with the output in the exact format specified in the system prompt, with no explanation or conversation.\n"
+)
+
+USER_DESIRE_DEFAULT_PROMOTE = (
```
Comment (waynehamadi) [inline autogpt/prompts/default_prompts.py]: DEFAULT_USER_DESIRE

File: autogpt/prompts/default_prompts.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    - Engage in effective problem-solving, prioritization, planning, and supporting execution to address your marketing needs as your virtual Chief Marketing Officer.
+
+    - Provide specific, actionable, and concise advice to help you make informed decisions without the use of platitudes or overly wordy explanations.
+
+    - Identify and prioritize quick wins and cost-effective campaigns that maximize results with minimal time and budget investment.
+
+    - Proactively take the lead in guiding you and offering suggestions when faced with unclear information or uncertainty to ensure your marketing strategy remains on track.
+    """)
+
+TASK_PROMPT_AICONFIG_AUTOMATIC = (
```
Comment (waynehamadi) [inline autogpt/prompts/default_prompts.py]: DEFAULT_TASK_PROMPT_AICONFIG_AUTOMATIC

File: autogpt/prompts/default_prompts.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+SYSTEM_PROMPT_AICONFIG_AUTOMATIC = ("""
```
Comment (waynehamadi) [inline autogpt/prompts/default_prompts.py]: DEFAULT_SYSTEM_PROMPT_AICONFIG_AUTOMATIC

File: autogpt/prompts/default_prompts.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    """)
+
+TASK_PROMPT_AICONFIG_AUTOMATIC = (
+    "Task: '{{user_prompt}}'\n"
+    "Respond only with the output in the exact format specified in the system prompt, with no explanation or conversation.\n"
+)
+
+USER_DESIRE_DEFAULT_PROMOTE = (
+    "Write a wikipedia style article about the project: https://github.com/significant-gravitas/Auto-GPT"
+)  # Default prompt
```
Comment (waynehamadi) [inline autogpt/prompts/default_prompts.py]: Create categories so people know where the constant is coming from.

Here it's 
#########################Setup.py#################################

File: autogpt/prompts/default_prompts.py (hunk 3 above)
Comment (waynehamadi) [reply autogpt/prompts/default_prompts.py]: RUN : 
black .
isort .

**Review by merwanehamadi — CHANGES_REQUESTED** [review]

File: autogpt/prompts/default_prompts.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+    - Proactively take the lead in guiding you and offering suggestions when faced with unclear information or uncertainty to ensure your marketing strategy remains on track.
+    """
+
+DEFAULT_TASK_PROMPT_AICONFIG_AUTOMATIC = (
+    "Task: '{{user_prompt}}'\n"
+    "Respond only with the output in the exact format specified in the system prompt, with no explanation or conversation.\n"
+)
+
+DEFAULT_USER_DESIRE_PROMOTE = "Write a wikipedia style article about the project: https://github.com/significant-gravitas/Auto-GPT"  # Default prompt
```
Comment (waynehamadi) [inline autogpt/prompts/default_prompts.py]: DEFAULT_USER_DESIRE_PROMOTE
=> DEFAULT_USER_DESIRE_PROMPT

**Review by merwanehamadi — CHANGES_REQUESTED** [review]

File: autogpt/prompts/default_prompts.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
+#########################Setup.py#################################
+
+DEFAULT_SYSTEM_PROMPT_AICONFIG_AUTOMATIC = """
```
Comment (waynehamadi) [inline autogpt/prompts/default_prompts.py:3]: run isort .

**Review by merwanehamadi — CHANGES_REQUESTED** [review]

**Review by merwanehamadi — APPROVED** [review]

**Comment by anonhostpi** [comment]
#3858

**Comment by anonhostpi** [comment]
#3990

**Pull request closed without merging**